src/main/java/io/percy/appium/metadata/AndroidMetadata.java

```diff
@@ -110,28 +107,7 @@ public String getDisplaySysDump() {
         return resultString;
     }
 
-
-    public String orientation() {
-      if (orientation != null) {
-          if (orientation.toLowerCase().equals("portrait") || orientation.toLowerCase().equals("landscape")) {
-              return orientation.toLowerCase();
-          } else if (orientation.toLowerCase().equals("auto")) {
-              try {
-                  return driver.getOrientation().toString().toLowerCase();
-              } catch (java.lang.NoSuchMethodError e) {
-                  return "portrait";
-              }
-          } else {
-              return "portrait";
-          }
-      } else {
-          Object orientationCapability = driver.getCapabilities().getCapability("orientation");
-          if (orientationCapability != null) {
-              return orientationCapability.toString().toLowerCase();
-          } else {
-              return "portrait";
-          }
-      }
-  }
-
+    protected ScreenOrientation driverGetOrientation() {
```

Comment:
`driverGetOrientation` name looks weird. Can we use something like getCurrentOrientation or checkOrientation ?

Reply:
We already have orientation function. Which is external ( public ) function. Both names suggested are close to that. 

So rather used driver prefix as it does driver.getOrientation and made it protected. 

Had to do cause appiumdriver does not have get orientation but android and ios driver does

Reply:
Open to any other better name though, which doesnt sound an alternative to orientation getter

Reply:
Np. As this is protected customer will not access it. We can go with the above name.